Tests/Test/ResponseTest.swift

```diff
@@ -7,12 +7,19 @@ class ResponseTest: XCTestCase {
     let response = Response()
     static let body200 = "<!DOCTYPE html><html><body><h1>Hello World</h1></body></html>"
     static let body404 = "<!DOCTYPE html><html><body><h1>404 Page Not Found</h1></body></html>"
-    let requestHeader = "\r\nCache-Control: no-cache\r\nConnection: keep-alive\r\n\r\n"
     let header200 = "HTTP/1.1 200 OK\r\nContent-Length: \(body200.utf8.count)\r\nContent-type: text/html\r\n\r\n"
     let header404 = "HTTP/1.1 404 Not Found\r\nContent-Length: \(body404.utf8.count)\r\nContent-type: text/html\r\n\r\n"
     let header200HEAD = "HTTP/1.1 200 OK\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
     let header404HEAD = "HTTP/1.1 404 Not Found\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
     
+    private func buildRequest(method: String, route: String, body: String="") -> String {
+        return "\(method) \(route)\r\nCache-Control: no-cache\r\nConnection: keep-alive\r\n\r\n\(body)"
+    }
+
+    private func buildResponse(statusCode: String, body: String) -> String {
+        return "HTTP/1.1 \(statusCode)\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\n\r\n\(body)" 
+    }
+
```

Comment:
These methods are a good start. I'm curious to hear what next steps you have in mind. :-)